src/gc/gc.cpp

```diff
@@ -3015,12 +3085,13 @@ void gc_heap::fire_pevents()
     settings.record (&gc_data_global);
     gc_data_global.print();
 
-    FIRE_EVENT(GCGlobalHeapHistory_V2, gc_data_global.final_youngest_desired, 
+    FireEtwGCGlobalHeapHistory_V2(gc_data_global.final_youngest_desired, 
```

Comment:
What is the reason for changing from FIRE_EVENT(GCGlobalHeapHistory_V2 to FireEtwGCGlobalHeapHistory_V2 ?

Reply:
all FireEtw* were changed to FIRE_EVENT for LocalGC because we wanna check if it's enabled on the GC side.